### PR TITLE
bug_5628 mako added

### PR DESCRIPTION
**Pull request by krzyhub**
https://bugzilla.stxnext.pl/show_bug.cgi?id=5628
https://gist.github.com/krzyhub/3fc8f95e7ec3cdfb0d3c

### CODE REVIEW

File: src/presence_analyzer/views.py [hunk 1]
```diff
@@ -2,10 +2,12 @@
 """
 Defines views.
 """
+#pylint: disable=no-name-in-module,import-error
 
 import calendar
-from flask import redirect, abort, render_template
+from flask import redirect, abort
```
Comment (mjelonek92) [inline src/presence_analyzer/views.py]: Please sort imports according to https://www.python.org/dev/peps/pep-0008/#imports

File: src/presence_analyzer/script.py [hunk 2]
```diff
@@ -6,9 +6,10 @@
 import sys
 from functools import partial
 
-import paste.script.command
 import werkzeug.script
 
+import paste.script.command
```
Comment (mjelonek92) [inline src/presence_analyzer/script.py]: Why you moved it?

File: src/presence_analyzer/tests.py [hunk 3]
```diff
@@ -1,14 +1,17 @@
 from __future__ import unicode_literals
+
+import datetime
+import json
+import os.path
+import unittest
+
+from presence_analyzer import main, utils, views
+
 # -*- coding: utf-8 -*-
```
Comment (mjelonek92) [inline src/presence_analyzer/tests.py]: This and docstring should be at the top.

File: src/presence_analyzer/views.py [hunk 4]
```diff
@@ -2,27 +2,26 @@
...
+
 from presence_analyzer.main import app
-from presence_analyzer.utils import(
-    jsonify,
-    get_data,
-    mean,
-    group_by_weekday,
-    seconds_since_midnight
+from presence_analyzer.utilsimport import (
+    get_data, group_by_weekday, jsonify,
```
Comment (mjelonek92) [inline src/presence_analyzer/views.py]: Each in separate line please.

File: src/presence_analyzer/views.py [hunk 5]
```diff
@@ -2,27 +2,26 @@
...
 )
 
-from jinja2 import TemplateNotFound
-from jinja2.exceptions import TemplateSyntaxError
-
-import logging
 log = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
+
```
Comment (mjelonek92) [inline src/presence_analyzer/views.py]: Why so many blank lines?

File: src/presence_analyzer/views.py [hunk 6]
```diff
@@ -132,7 +134,7 @@ def render_html(template):
         }
     try:
         return render_template(template + '.html', urls=urls)
-    except TemplateSyntaxError:
+    except exceptions.TopLevelLookupException:
```
Comment (mjelonek92) [inline src/presence_analyzer/views.py]: Merge those two except statements into one please.

File: src/presence_analyzer/views.py [hunk 7]
```diff
@@ -132,7 +134,7 @@ def render_html(template):
         }
```
Comment (mjelonek92) [inline src/presence_analyzer/views.py]: Style.

File: src/presence_analyzer/script.py (hunk 2 above)
Comment (krzyhub) [inline src/presence_analyzer/script.py]: isort script.py

File: src/presence_analyzer/views.py (hunk 4 above)
Comment (krzyhub) [inline src/presence_analyzer/views.py]: isort views.py work

File: src/presence_analyzer/views.py (hunk 6 above)
Comment (krzyhub) [inline src/presence_analyzer/views.py]: Your solution with merging this two by putting them in brackets does not working and last time using multiple 'except' was okay - I don't know why now it is a problem

**Comment by dpolcyn** [comment]
```ImportError: No module named flask.ext.mako```

**Comment by mjelonek92** [comment]
Cool :+1:, merge.

**Pull request merged**